translations/i18nCheck.R

```diff
@@ -9,45 +9,72 @@ checkStatus <- c()
 cli_h1("Check R translations:")
 rPotData <- potools::get_message_data(dir = ".", verbose = TRUE)
 
-# Get wrong usage of gettext from .R
-rErrorCalls <- subset(rPotData, grepl(pattern="gettext(|f)\\(['\"]['\"]\\)", call), select = c("file", "call", "line_number"))
-          
-if (nrow(rErrorCalls) > 0) {
+# Get unreasonable usage of gettext from .R
+placeholderData <- subset(rPotData,
+                          grepl(pattern = "(.*%[a-zA-Z].*|.*%[0-9]*[a-zA-Z].*){2,}", call) & !grepl(pattern = "%%", call),
+                          select = c("file", "call", "line_number"))
+rEmptyCalls <- subset(rPotData,
+                      grepl(pattern = "gettext(|f)\\(['\"]['\"]\\)", call),
+                      select = c("file", "call", "line_number"))
+
+# Get po/mo compiling error
+e <- capture.output(tools::update_pkg_po("."))
+
+if (length(e) > 0) {
+  checkStatus <- c(checkStatus, 1)
+  # Format data to make read friendly in Github CL
+  rPoError <- as.data.frame(matrix(e, ncol=5, byrow=TRUE))[1:4]
+  colnames(rPoError) <- c("Error_Location", "Eorr_Type", "Original_Gettext", "Translated_text")
```

Comment:
```suggestion
  colnames(rPoError) <- c("Error_Location", "Error_Type", "Original_Gettext", "Translated_text")
```
I'm not sure but this looks like a typo?

Reply:
Ah yep, a typo😛